api/client/node/update.go

```diff
@@ -47,7 +51,6 @@ func runUpdate(dockerCli *client.DockerCli, nodeID string, mergeNode func(node *
 		return err
 	}
 
-	fmt.Fprintf(dockerCli.Out(), "%s\n", nodeID)
```

Comment:
I see. The pattern with the cli functions was to have most of the logic that would have been in the `RunE()` extracted into the `runX()` functions, but those `runX()` weren't really designed to be re-used. I guess it's not a big deal. We would need to refactor this further to support that idea.